modules/ROOT/pages/policy-mule4-detokenization.adoc

```diff
@@ -21,15 +21,9 @@ endif::[]
 
 |===
 
-[NOTE]
---
-This policy is designed to work only for Anypoint Runtime Fabric deployments.
---
 
-== Detokenization
+Detokenization is the process of detokenizing a token back into its original value to redeem sensitive data for fulfilling specific business needs. For example, an application may need to use a detokenized value to validate a specific credit card number. *<Eva, can we add a better example?>*
```

Comment:
Tokenization is the process of replacing sensitive information with tokens to minimize the exposure of the original data and reduce the risk of compromising this information. Applications can operate with the tokens instead of the original data and, when there is an exception and the application requires access to the sensitive data, the detokenization policy allows to redeem this data. For example, for business purposes, an application might need to know the bank account number or the credit card number to process it.

Reply:
@evangelina is this for Tokenization or Detokenization?

Reply:
Context on detokenization and the use case

Reply:
Done! Thanks!